lib/oban.ex

```diff
@@ -185,9 +185,24 @@ defmodule Oban do
     children = children ++ Enum.map(plugins, &plugin_child_spec(&1, conf))
     children = children ++ Enum.map(queues, &QueueSupervisor.child_spec(&1, conf))
 
+    execute_init(conf)
+
     Supervisor.init(children, strategy: :one_for_one)
   end
 
+  defp execute_init(conf) do
+    measurements = %{
+      system_time: System.system_time()
+    }
+
+    metadata = %{
+      pid: self(),
+      config: conf
+    }
```

Comment:
The data looks good, but let's tighten up the spacing in this already large module:

```suggestion
    measurements = %{system_time: System.system_time()}
    metadata = %{pid: self(), config: conf}
```